dsymbol/src/dsymbol/ufcs.d

```diff
@@ -2,15 +2,31 @@ module dsymbol.ufcs;
 
 import dsymbol.symbol;
 import dsymbol.scope_;
+import dsymbol.builtin.names;
+import dsymbol.utils;
+import dparse.lexer : tok, Token;
 import std.functional : unaryFun;
 import std.algorithm;
 import std.array;
 import std.range;
-import dsymbol.builtin.names;
 import std.string;
-import dparse.lexer : tok;
 import std.regex;
 import containers.hashset : HashSet;
+import std.experimental.logger;
+
+enum UFCSCompletionContext
+{
+    DotCompletion,
+    ParenCompletion,
+    UnknownCompletion
```

Comment:
make UnknownCompletion the first item, so it's the default value for this enum type